Resources/Prototypes/Entities/Objects/Materials/materials.yml

```
@@ -380,6 +380,14 @@
   - type: Stack
     count: 1
 
+- type: entity
+  parent: MaterialDiamond
+  id: MaterialDiamond5
+  suffix: five
+  components:
+  - type: Stack
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)